connector-packager/tests/test_xsd_validator.py

```diff
@@ -65,3 +65,21 @@ def test_validate_single_file(self):
         logging.debug("test_validate_single_file xml violations:")
         for violation in xml_violations_buffer:
             logging.debug(violation)
+
+    def test_validate_vendor_prefix(self):
+
+        test_file = TEST_FOLDER / Path("modular_dialog_connector/connectionFields.xml")
+        file_to_test = ConnectorFile("connectionFields.xml", "connection-fields")
+        xml_violations_buffer = []
+
+        self.assertTrue(validate_single_file(file_to_test, test_file, xml_violations_buffer),
+                        "Valid XML file not marked as valid")
+
+        print("\nTest malformed xml. Throws XML validation error.")
+        test_file = TEST_FOLDER / Path("broken_xml/connectionFields.xml")
+        self.assertFalse(validate_single_file(file_to_test, test_file, xml_violations_buffer),
```

Comment:
Could we use `[assertRaises](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises)` here instead?

Reply:
This is the pattern used by the other tests for connector-packager.  I'll make a note to update all at once later.